pmlc/dialect/pxa/transforms/trunci_transform.h

```diff
@@ -0,0 +1,13 @@
+//
+// Created by liyangling on 8/20/20.
+//
+
+#pragma once
+
+#include "mlir/Dialect/StandardOps/IR/Ops.h"
+
+namespace pmlc::dialect::pxa {
+
+mlir::TruncateIOp truncITypeTransformation(mlir::TruncateIOp op);
```

Comment:
Can we add this pattern conversion to the i1 to i32 pass?

Reply:
Sure, I'll do it

Reply:
Done. Please help review.
Current full test result is 
`FAILED (failures=4, errors=35, skipped=10, expected failures=1)`

Reply:
Thanks. It looks good to me. Could you please add the one hot test to pmlc/dialect/stdx/tests/i1_to_i32.mlir ?